src/citrine/resources/file_link.py

```diff
@@ -639,14 +588,14 @@ def _complete_upload(self, dest_name: str, uploader: _Uploader):
                                                       json={'s3_version': uploader.s3_version})
 
         try:
+            print(complete_response['file_info'])
```

Comment:
do you want to keep this print statement?

Reply:
No 🙇